scripts/js/common.js

```diff
@@ -324,6 +324,13 @@ function addAudio(obj, cont, rootPath, containerType) {
         desc = artist_full;
     }
 
+  
+    var albumArtist = artist[0];
```

Comment:
It has to be an array of strings. So either
`albumArtist = artist` or `albumArtist = [ artist[0] ]`